topbeat/beat/sigar.go

```diff
@@ -15,113 +15,41 @@ type SystemLoad struct {
 }
 
 type CpuTimes struct {
-	User          uint64  `json:"user"`
+	Cpu           sigar.Cpu
```

Comment:
I'm wondering if it would be better to embed `sigar.Cpu` in the `CpuTimes`. That should remove the stutter of `CpuTimes.Cpu`. The same applies to mem and swap.